client/app/services/auth-guard-login.service.ts

```diff
@@ -8,7 +8,11 @@ export class AuthGuardLogin implements CanActivate {
   constructor(public auth: AuthService, private router: Router) {}
 
   canActivate() {
-    return this.auth.loggedIn;
+    if (this.auth.loggedIn()) {
```

Comment:
you can shrink this: `return this.auth.loggedIn();`

Reply:
It's ok, mabye just add a comment stating that a redirection would fit well in those lines.